benefits/static/css/styles.css

```diff
@@ -556,7 +564,7 @@ footer .footer-links li a.footer-link:visited {
 .form-text {
   font-size: var(--font-size-14px);
   line-height: var(--form-field-text-line-height);
-  letter-spacing: var(--letter-spacing-5);
+  letter-spacing: calc(var(--font-size-14px) * var(--letter-spacing-5));
```

Comment:
<img width="577" alt="image" src="https://github.com/user-attachments/assets/fd69e693-a7e4-4486-adb6-52e2b05cc815" />
This letter-spacing update affects not just reCAPTCHA text, but the form field helper text as well. It was not updated on Production.

The way this is calculated: In Figma, click on the text. In the Dev resources sidebar, ensure Properties is enabled (instead of Code). Look for the letter spacing percentage:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/1dfc495d-2d48-46dd-b174-3f6971b3a249" />

Multiply the percentage with the font size.